### PR TITLE
Setup Portfolio Project

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
Project for mobile use first. In this project a template is chosen from figma and made its mobile responsive version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <li><a href="#" id="linkedin">
+            <img src="Images\linkedin.png" alt="linkedin"></a></li>
+        <li><a href="#" id="angelist">
+            <img src="Images\angel.png" alt="angelist"></a></li>
+        <li><a href="#" id="twitter">
+            <img src="Images\twitter.png" alt="twitter"></a></li>
+        <li><a href="#" id="medium">
+            <img src="Images\medium.png" alt="medium"></a></li>
+      </ul>
+    </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The container of this section is a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info 🙃 🙃

File: style.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+#twitter-icon {
+  order: 3;
+  border-radius: 8px;
+}
+
+#medium-icon {
+  order: 4;
+  border-radius: 8px;
+}
```
Comment (cynthiainga) [inline style.css]: - [ ] Please kindly make sure your design stick as much as possible to the Figma design. You have some spaces to the left and to the top of your page, kindly fix it 🙃 🙃 

Your design

Screen Shot 2022-10-28 at 22 31 40

Figma design

Screen Shot 2022-10-28 at 22 42 50

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Talha-Hanif5666, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  margin-left: 24px;
+}
+
+.primary-text {
+  width: 327px;
+  height: 124px;
+  display: flex;
+  margin-left: 24px;
+  margin-top: 120px;
+  font-family: 'Crete-Round', sans-serif;
```
Comment (aimalamiri) [inline style.css]: Good job for using the proper font name, but you forget to include the font in your `head` tag. That's why this font is not applied. You can take it from the [google fonts](https://fonts.google.com/specimen/Crete+Round?query=Crete+Round).

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @Talha-Hanif5666, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights

- The header section is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
# Hi @Talha-Hanif5666  😄 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 



## Highlights
The commit messages are meaningful ✅
Linters are working properly ✅
Github flow observed ✅ 

## Optional suggestions

- [ ] [optional] Please kindly take note that your commit messages should be in sentence case.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @oyepriscilla in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**